### ISSUE TITLE
Add config var to only specify what to do when an esil emulation exception happens

### ISSUE THREAD

**Issue by radare** [radare/radare2#2823]
* ignore and continue
* stop emulating there
* ...

ESIL exceptions should happen when trying to emulate an instruction that doesnt have an ESIL representation (marked as TODO). or it's just an invalid instruction.

Other kind of ESIL exceptions may happen when trying to read or write on non-allocated memory. This case is not handled yet. and we should have C callbacks for this case. /cc @condret

**Comment by ret2libc** [comment]
Is this still relevant?

**Comment by trufae** [comment]
Yep. and its related to the dr/aer/ar dilema about having emulation in sync with debugging experience

**Comment by ret2libc** [comment]
This issue has been moved from radareorg/radare2 to radareorg/ideas as we are trying to clean our backlog and this issue has probably been created a long while ago. This is an effort to help contributors understand what are the actionable items they can work on, prioritize issues better and help users find active/duplicated issues more easily. If this is not an enhancement/improvement/general idea but a bug, feel free to ask for re-transfer to main repo. Thanks for your understanding and contribution with this issue.
